sepa/src/main/java/com/adyen/checkout/sepa/Iban.kt

```diff
@@ -0,0 +1,290 @@
+/*
+ * Copyright (c) 2017 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by timon on 13/08/2017.
+ */
+package com.adyen.checkout.sepa
+
+import java.math.BigInteger
+import java.util.Arrays
+import java.util.Collections
+import java.util.Locale
+import java.util.regex.Pattern
+
+class Iban private constructor(val value: String) {
+
+    @Suppress("MemberVisibilityCanBePrivate")
+    val countryCode: String = value.substring(0, COUNTRY_CODE_POSITION_END)
+    val checkDigits: String = value.substring(CHECK_DIGIT_POSITION_START, CHECK_DIGIT_POSITION_END)
+    @Suppress("unused")
+    val bban: String
```

Comment:
This could just be `val bban: String = value.substring(IBAN_BLOCK_SIZE)` as the value of `value` can't change